File: source/test/modelBuilder/res/userModelWithArray.java
package com.lotus.tn.model;

import java.util.ArrayList;

public class User {
    private static final String TAG = "User";

    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    private Integer age;

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    private String gender;

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    private ArrayList<String> address;

    public ArrayList<String> getAddress() {
        return address;
    }

    public void setAddress(ArrayList<String> address) {
        this.address = address;
    }
}

File: source/test/modelBuilder/res/userModelWithArray.js
/**
 * Created by danney on 16/4/23.
 */

module.exports = {
    import: [
        'java.util.ArrayList'
    ],

    name: 'User',
    properties: [
        {name: 'name', type: 'string'},
        {name: 'age', type: 'int'},
        {name: 'gender', type: 'string'},
        {name: 'address', type: 'ArrayList<String>'}
    ]
}